nixos/modules/services/backup/restic.nix

```diff
@@ -254,18 +272,22 @@ in
           } // optionalAttrs (backup.s3CredentialsFile != null) {
             EnvironmentFile = backup.s3CredentialsFile;
           };
-        } // optionalAttrs (backup.initialize || backup.dynamicFilesFrom != null) {
+        } // optionalAttrs (backup.initialize || backup.preStart != null || backup.dynamicFilesFrom != null) {
           preStart = ''
+            ${builtins.toString backup.preStart}
```

Comment:
don't we need to use `optionalString` here, to prevent a literal `"null"` from appearing?